quality/fca/cypress/integration/usager/hybridge.feature

```diff
@@ -0,0 +1,21 @@
+#language: fr
+Fonctionnalité: Hybridge
+
+  @ignoreDocker
+  Scénario: Test de l'hybridge en intégration
+    Etant donné que je navigue sur la page fournisseur de service
+    Et que je clique sur le bouton ProConnect
+    Et que j'entre l'email "should-call-@fia3-rie.fr"
+    Quand je clique sur le bouton de connexion
+    Et je suis redirigé vers la page permettant la selection d'un fournisseur d'identité
+    Et je teste l'hybridge avec le fournisseur d'identité "Identity Provider 3 - RS256 RIE"
+
+  @ignoreInteg01 @hybridge
```

Comment:
Suggestion: I find the use of the @hybridge annotation hard to understand.

To summarize, @hybridge is used to run tests on the Hybridge stack, but only locally. It is used in conjunction with @ignoreInteg01. In integration environments, this annotation has no effect, so there's no need to use it alongside @ignoreDocker.

I would prefer to apply the @hybridge annotation at the file level.

Reply:
:thinking: We definitionally need to clarify this tagging politic...